bitrisescript/src/bitrisescript/task.py

```diff
@@ -1,33 +1,33 @@
-from textwrap import dedent
 from typing import Any
 
 from scriptworker_client.exceptions import TaskVerificationError
 from scriptworker_client.utils import get_single_item_from_sequence
 
 
-def validate_scope_prefixes(config, task):
-    """Validates scope prefixes.
+def _get_allowed_scope_prefixes(config):
+    prefixes = config["taskcluster_scope_prefixes"]
+    return [prefix if prefix.endswith(":") else "{}:".format(prefix) for prefix in prefixes]
+
+
+def extract_common_scope_prefix(config: dict[str, Any], task: dict[str, Any]) -> str:
```

Comment:
are these scope resolution functions generic enough to move to something like scriptworker_client.task?

Reply:
I think so, good call! I copy/pasted it from another script and it looks like a bunch of them have something very similar.. Should be a follow-up though.